src/lib/analytics/apis.native.js

```diff
@@ -1,7 +1,15 @@
 import * as SentryNative from '@sentry/react-native'
 import amplitude from 'amplitude-js'
+import analytics from '@react-native-firebase/analytics'
+
+const googleAnalyticsWrapper = {
+  push: ({ event, ...data }) => {
+    analytics().logEvent(event, data)
+  },
+}
 
 export default {
   sentry: SentryNative,
   amplitude: amplitude.getInstance(),
+  googleAnalytics: googleAnalyticsWrapper,
```

Comment:
3.
```suggestion
  googleAnalytics: new GoogleWrapper(analytics()),
```

4. add the following `GoogleWrapper` to `apis.web`:

```javascript
import { convertToGoogleAnalytics } from './util'
class GoogleWrapper {
  logEvent(event, data: any = {}) {
    const eventData = convertToGoogleAnalytics(data)
    
    dataLayer.push({ event ...eventData })
  }
}
```

5. adjust the export in `apis.web`:

```javascript
googleAnalytics: dataLayer ? new GoogleWrapper() : null,
```

6. remove `convertToGA` from analytic class, adjust `fireGoogleAnalyticsEvent`